src/libraries/Fuzzing/DotnetFuzzing/Fuzzers/UTF8Fuzzer.cs

```diff
@@ -0,0 +1,155 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Buffers;
+using System.Text.Unicode;
+using System.Text;
+
+namespace DotnetFuzzing.Fuzzers;
+
+// Adapted from https://github.com/GrabYourPitchforks/utf8fuzz
+internal sealed class UTF8Fuzzer : IFuzzer
+{
+    public string BlameAlias => "mizupan";
```

Comment:
Feels unsustainable to be assigning these to a specific alias. Can we create some kind of team alias that these are assigned to instead?

Reply:
Sadly can't assign work items to groups / multiple people in AzDo.
I've changed it to always assign me for now, and I'll re-assign as needed (we shouldn't be getting these reports all that often anyway).